test/test_jit.py

```diff
@@ -10565,6 +10565,89 @@ def addmm_grad_test(b, x, w):
         self.assertEqual(w.grad, w_ref.grad)
         self.assertEqual(b.grad, b_ref.grad)
 
+    def test_layer_norm_grad(self):
```

Comment:
1. Should we confirm that layer_norm was indeed auto differentiated?
Also
this test looks very similar to the tests `jit_metaprogramming_utils.py` should be doing:
* fwd result check against ref
* grad check against grad_ref
* grad grad check against grad grad ref

Is there a reason for a separate test?

Reply:
IIRC, the existing test in `jit_metaprogramming_utils.py` doesn't check for all the combination of `weight` / `bias` / `training`. We have this test for better coverage.

But you are absolutely right that there's no good reason to keep them. I can try to come up with a similar test using the meta test infrastructure.

Reply:
I double checked the `jit_metaprogramming_utils.py`, layer_norm does seem to get a good combination of weight/bias there. The problem I mentioned earlier only applies to batch_norm. I'll remove the test here.